### DISCUSSION TITLE
Rustdesk on 22.04 aarch64 Raspberry Pi

### DISCUSSION THREAD

**Discussion by alphafox02** [rustdesk/rustdesk#8952] [Q&A] [answered]
### Bug Description

Attempting to install Rustdesk 1.2.7 aarch64 deb will prompt a user that libayatana-appindicator3-1 and network-manager-gnome and blueman must first be uninstalled. I believe this is due to Rustdesk 1.2.7 needed libayatana-appindicator3-1 installed which apparently conflicts with libayatana-appindicator3-1.

### How to Reproduce

Install Ubuntu 22.04 on Raspberry Pi and attempt to install 1.2.7 RustDesk.

### Expected Behavior

I expect the network manager not to be removed, which may mean rustdesk could maybe use libayatana-appindicator3-1? 

### Operating system(s) on local side and remote side

22.04 Ubuntu Aarch64

### RustDesk Version(s) on local side and remote side

1.2.7

### Screenshots

NA

**Comment by alphafox02** [comment]
As a follow up, this does not occur on 22.04 x86_64 with the rust desk 1.2.7 deb package and what is interesting is that the system has libayatana-appindicator-3-1 installed.

Even though the control file of rustdesk 1.2.7 says it depends on libappindocator3-1 it installs fine.

**Comment by alphafox02** [comment]
I edited the aarch64 rustdesk deb and removed libappindocator3-1 from required and it installs fine and runs without removing all other packages I mentioned. I’m not sure why the difference between asrch64 and x86_64

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/pull/9364

**Reply by alphafox02** [reply]
Awesome. I’ll try this asap.
